client/src/pages/Landing/LandingLoggedIn.vue

```diff
@@ -62,7 +62,7 @@ export default {
         query: gql`
           query GetProjectsByUserId($userId: Int!) {
             assets(
-              where: {permissionset: {permissions: {group: {groups_users: {user_id: {_eq: $userId}}}}}},
+              where: {asset_type: {_eq: project}, _and: {created_by_id: {_eq: 1}}}
```

Comment:
Your permissions will only show projects where the user created the project. The previous code uses the more complex permissionset logic--all of that, however, may not have been added back into Hasura to make it work. We can ignore this for now, I can go back in and make sure permissionset logic is added back in, or I can walk you through it at some point.

Reply:
Sounds good, the query with permissionset doesn't work that is why I changed to the current one. We can keep temporarily until it is fixed.